### DISCUSSION TITLE
I want to make selected text bold/italics/underline/highlight/hyperlink in the CTkTextBox Widget... Is that even possible?? If so please please guide me on how to do so 🙏🏻🙏🏻

### DISCUSSION THREAD

**Discussion by Zingzy** [TomSchimansky/CustomTkinter#1897] [General]

**Comment by abhaymakes** [comment]
Hello @Zingzy, this effect can be achieved by using [tags](https://tkdocs.com/tutorial/text.html) inside tkinter as well as customtkinter.

For formatting the text(bold, italic etc), have a look at this [tutorial](https://www.youtube.com/watch?v=X6zqePBPDVU).
